### ISSUE TITLE
function to convert ZH graphs to ZX

### ISSUE THREAD

**Issue by mhlr** [Quantomatic/pyzx#77]
In particaular is there a way to decompose higher order Hadamard nodes into Z, X and binary H?

`zx.hsimplify.from_hypergraph_form` doc specifically says:
> Note that more general hboxes are not changed, so this does *not* give a ZX graph in general.

**Comment by jvdwetering** [comment]
There is currently nothing implemented in PyZX that allows you to do this. If you are interested in doing something like this, how to translate H-boxes into ZX is described in https://arxiv.org/abs/1904.07551.
Note that the standard translation does mean that the decomposition of an n-ary H-box will involve spiders with phases of 2pi/2^n.

**Comment by mhlr** [comment]
Thanks Jan! I had not seen that paper I was mainly working through the counting & satisfiability paper.

What is teh right place to ask more general pyzx questions, not just issues & bugs?

**Comment by jvdwetering** [comment]
There is a ZX-calculus Discord server, where you can ask whatever ZX-related questions you want. You can find an invite link on the ZX-calculus website: http://zxcalculus.com/

**Comment by banrovegrie** [comment]
Is there any update on this? Else maybe I could take this up?

**Comment by jvdwetering** [comment]
@banrovegrie Sorry for not getting back at you about this. If you want to implement something like this, that would be great! @akissinger and I were actually thinking of making this an issue with a bounty for the unitaryHACK hackathon

**Comment by banrovegrie** [comment]
Yeah, it will be fun to work on it as a part of unitaryHACK as well.

**Comment by jvdwetering** [comment]
Closing this as #92 supercedes this
